cypress/integration/cluster_spec.js

```diff
@@ -0,0 +1,23 @@
+describe('Cluster functionality test', () => {
+  it('adds cluster with valid data', () => {
+    cy.visit('/');
+    cy.get('#add-cluster-btn').then(() => {
+      cy.get('#add-cluster-btn')
+        .first()
+        .click()
+        .then(() => {
+          cy.get('#cluster-name-input')
+            .type('test cluster')
+          cy.get('#url-input')
+            .type('example.com')
+          cy.get('#token-input')
+            .type('123456')
+          cy.get('#check-connection-btn')
+            .click()
+          cy.get('#submit-cluster-btn')
+            .click()
+        })
+    });
+    cy.contains('Add cluster')
```

Comment:
need some sort of check to see that the cluster has actually been added. This line was a placeholder until that logic was sorted out.

Reply:
Added check cluster by name